Washington-County/COUNTY-DRUPAL
# WCOR-1085: Add a folder in Drupal's docroot to house static microsites and applications

**Use case**

As a site administrator, I need a central place to host approved microsites and web applications that supplement the Washington County website.

**Implementation**

1. Create the following directory: /docroot/apps
2. Create /docroot/apps/.htaccess following the guidance of these tutorials:

https://support.acquia.com/hc/en-us/articles/360006384074-Maintaining-a-static-microsite-alongside-Drupal-s-docroot
https://drupalninja.wordpress.com/2018/04/09/how-to-host-a-microsite-inside-a-drupal-docroot-hosted-in-acquia/

**Test plan**

1. Create file /apps/index.php. Ensure that index.php renders and can be accessed as an anonymous user.
2. Check that index.php loads by default when browsing to /apps/.
3. Create an HTML file under /apps/. Check that the HTML file can be accessed as an anonymous user.
4. In /apps/, create a subdirectory containing a PHP file. For example, /apps/test/abc.php. Check that the PHP file in the subdirectory can be accessed as an anonymous user.
